bin/autocomplete

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+
+class AutoComplete
+  def initialize(input = ARGV[0])
+    h = eval(input)
```

Comment:
Sorry that commit should have been on a different branch to this one. Removed it now.

But in relation to your question, it converts the string to a ruby `hash`. Will look into those errors in the current branch on working on it